packages/report-server/src/types.ts

```diff
@@ -3,51 +3,21 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { Request } from 'express';
-import { ParamsDictionary, Query } from 'express-serve-static-core';
-import { AccessPolicy } from '@tupaia/access-policy';
-import { TupaiaDatabase, ModelRegistry } from '@tupaia/database';
+import { ModelRegistry } from '@tupaia/database';
+import { ReportModel } from './models';
 
-export interface FetchReportFilter {
+export interface ReportServerModelRegistry extends ModelRegistry {
+  readonly report: ReportModel;
```

Comment:
Super happy that now the report-server only has a dependency on the `Report` model! 👍